services/brig/test/resources/turn/servers-v2.txt

```diff
@@ -1,2 +1 @@
-turn:localhost:3478
-turn:localhost:3479
\ No newline at end of file
+turn:localhost:3478
```

Comment:
Apparently this file is modified like this automatically whenever I run integration tests. Or at least sometimes.

Is this okay?

Reply:
it's debatable whether this is ok, but it's expected behavior.  if you want to fix it, do it elsewhere!  :-)

Reply:
(fviw, i would change the tests so they never change files under version control.  if they want to write to something that is under version control, they should copy it to another place first that is covered by .gitignore, and write to it there.)